README.md

```diff
@@ -606,25 +606,26 @@ Which database backend to use for the read database. Only supports
 `postgres` (default). This option is supported in PuppetDB >= 1.6.
 
 #### `read_database_host`
-*This parameter must be set to enable the PuppetDB read-database.*
+*This parameter must be set to use another PuppetDB instance for queries.*
 
-The hostname or IP address of the read database server. Defaults to `undef`.
-The default is to use the regular database for reads and writes. This option is
+The hostname or IP address of the read database server. If set to `undef`, 
+it will use the value of the `database_host` parameter. This option is
 supported in PuppetDB >= 1.6.
 
 #### `read_database_port`
 
-The port that the read database server listens on. Defaults to `5432`. This
+The port that the read database server listens on. If `read_database_host`
+is set to `undef`, it will use the value of the `database_port` parameter. This
 option is supported in PuppetDB >= 1.6.
 
 #### `read_database_username`
 
-The name of the read database user to connect as. Defaults to `puppetdb`. This
+The name of the read database user to connect as. Defaults to `puppetdb-read`. This
```

Comment:
Does this always default to `puppetdb-read`. If `managed` is set to false will it still default to `puppetdb-read` or will it default to `database_username`?

Reply:
This always defaults to puppetdb-read